src/core/Akka.Persistence.Tests/AtLeastOnceDeliveryFailureSpec.cs

```diff
@@ -144,9 +144,9 @@ public ChaosSender(IActorRef destination, IActorRef probe)
                 _config = Context.System.Settings.Config.GetConfig("akka.persistence.sender.chaos");
                 _liveProcessingFailureRate = _config.GetDouble("live-processing-failure-rate");
                 _replayProcessingFailureRate = _config.GetDouble("replay-processing-failure-rate");
-            }
 
-            public override TimeSpan RedeliverInterval { get { return TimeSpan.FromMilliseconds(500); } }
+                RedeliverInterval = TimeSpan.FromMilliseconds(500);
```

Comment:
This is a good example how these properties are now intended to be used; just set them anywhere from within the actor instead of overwriting.